channels/us.m3u

```diff
@@ -1740,6 +1738,8 @@ https://gustous-plex.amagi.tv/hls/amagi_hls_data_gustoAAAA-gustous-plex/CDN/play
 https://dai.google.com/linear/hls/event/tdSCy5u2R5WtCLXX4NwDtg/master.m3u8
 #EXTINF:-1 tvg-id="GunAzTV.us" tvg-name="GünAz TV" tvg-country="US;AZ" tvg-language="Azerbaijani" tvg-logo="" group-title="General",GünAz TV
 rtsp://gtv.live-s.cdn.bitgravity.com/cdn-live/_definst_/gtv/live/feed02?nc=1
+#EXTINF:-1 tvg-id="GunAzTV.us" tvg-name="GünAz TV" tvg-country="US;AZ" tvg-language="Azerbaijani" tvg-logo="" group-title="General",GünAz TV
+http://gtv.live.cdn.bitgravity.com/gtv/live/feed03
```

Comment:
Hmm... why do I get a 1gb flv file on this one ?

Reply:
Looks like a legit broadcast to me

Reply:
It just sounds weird to get a fixed file. I'm gonna take a look.